openapi/openapi/src/main/scala/endpoints4s/openapi/model/OpenApi.scala

```diff
@@ -89,14 +89,27 @@ object OpenApi {
         }
         obj.additionalProperties.foreach(p => fields += "additionalProperties" -> schemaJson(p))
       case array: Schema.Array =>
-        val itemsSchema = array.elementType match {
-          case Left(value)  => schemaJson(value)
-          case Right(value) => ujson.Arr(value.map(schemaJson): _*)
+        fields += "type" -> "array"
+        array.elementType match {
+          case Left(value) =>
+            fields += "items" -> schemaJson(value)
+          case Right(value) =>
+            /** Best effort to represent the heterogeneous array in OpenAPI 3.0
+              * This should be changed with OpenAPI 3.1 and more idiomatic representation using `prefixItems`
+              */
+            fields ++= List(
+              "items" -> schemaJson(
+                Schema.OneOf(
+                  alternatives = Schema.EnumeratedAlternatives(value),
+                  description = None,
```

Comment:
I was thinking of maybe using the description here to stress additional constraint (fixed positions of elements in a tuple), that cannot be enforced by OpenAPI 3.0 features (along the lines of [this example](https://stackoverflow.com/a/57465199/15821235)), but I'm not sure what would be an appropriate format/language.

Reply:
I think it’s fine to leave it empty. What do others think?